cv_next/app/actions/cvs/fetchCvs/route.ts

```diff
@@ -0,0 +1,22 @@
+"use server";
```

Comment:
We should keep things separates; `actions` folder is for server actions, api routes should be under a different folder - e.g. in a different `api` folder